.ci/Jenkinsfile-extension-vsce

```
@@ -100,15 +100,29 @@ pipeline {
                             git merge -X theirs issue/version-bump-extension
 
 
+                            if [ -f "${WORKSPACE}/requirements.txt" ]; then
```

This is not correct, is it. We are in the sub-directory `vscode-inmanta`. Checking at `vscode-inmanta` would mean that we check for the file outside of the vscode-inmanta repository.
```suggestion
                            if [ -f "requirements.txt" ]; then
```

---

oups indeed, sorry and thanks! I went to fast